src/main/index.html

```diff
@@ -2,37 +2,31 @@
 <html lang="en">
 <head>
 <meta charset="utf-8" />
-<title>TMDB API Programming Test</title>
+<title>Awesome TMDB App</title>
 
 <!-- Default Styles (DO NOT TOUCH) -->
 <link rel="stylesheet" type="text/css" href="/css/app.css">
 <!-- RequireJS // -->
 <script src="/vendor/requirejs/require.js" data-main="main"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+<style type="text/css">
+		html
+			{
+			 font-size: 14px;
+			}
+		
+
+</style>
 </head>
 <body>
 	<div class="wrapper">
-		<div class="navbar nav">
-			<div class="container-fluid">
-				<div style="position: absolute; top: 0; left: 0; z-index: 50; font-size: 24px; margin-top: 24px; margin-left: 20px;">
-					<a href="#/" title="Start Over">TMDB API Programming Test</a>
-				</div>
-				<div class="pull-right" style="margin-top: 21px;">
-            		<search />
-        		</div>
-			</div>
-		</div>
-		<div class="contain">
-			<!-- BEGIN PAGE CONTENT -->
-			<div class="content">
+		<ng-view></ng-view>
+	</div>
 
-				<div protected-content>
 
-					<ng-view></ng-view>
-				</div>
-			</div>
-			<!-- END PAGE CONTENT -->
-		</div>
-	</div>
+
+
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
There are two inclusions of jquery here -- do you know which one will actually be used?